#3 nodejs server example

JoelPfammatter98 opened the pull request:
## Description
Created a small server with nodejs and typescript. To start the build run "npm install"  once and then 
"npm start" to compile and run the application. Pls make sure to enter the nodeAPI folder first!

JoelPfammatter98 committed:
nodeserver

bregySilvan commented:
@TheScorpio13 
When creating a PR(pull request) you should provide a description what you've actually done. Even with just 1 commit.  It's helpful to for reviewing. 
You should also connect the PR with the matching Issue.

@olinator98 
Pls do  review and test.

bregySilvan committed:
#3 add start command

-Added a command for compiling and starting at once. It can be called via "npm start".
Before the first run, you should also call "npm install" once.

bregySilvan reviewed (approved):
Nice Job Joel!
With this basic functionality we can start to build our API.
I think it would be nice if we had live compiling. We should discuss in the next meeting.

Christian-Seematter commented on exemples/nodeAPI/app.ts:
```diff
@@ -0,0 +1,44 @@
+
+      // Configure API endpoints.
+      private routes(): void {
+        /* This is just to get up and running, and to make sure what we've got is
```
Now that you understand commits and Pull Requests, this is the type of comment that no longer needs to be in the source. The problem here is it might stay for longer than needed and gives wrong information later in the project. It's also not necessarily visible to the people using the server.

Christian-Seematter reviewed (approved):
Good work! This is a great setup to start with and build on top of. 

When you create the PR, make sure you give an overview of the changes made, run instructions and info about technical changes made. For example the fact that a package-lock is introduced for this demo,  that it should return a simple HTML demo page and that it runs on port 3000.
